app/src/common/urls.js

```diff
@@ -129,6 +129,8 @@ export const URLS = {
     `${urlCommonBase}organizations${getQueryParams(preferencesObj)}`,
   organizationProjects: (organizationId, preferencesObj = {}) =>
     `${urlCommonBase}organizations/${organizationId}/projects${getQueryParams(preferencesObj)}`,
+  filterOrganizationProjects: (organizationId) =>
```

Comment:
May be adjust the name to be more consistent with endpoint name?
e.g. `organizationProjectsSearches`